libs/scully/src/scully.ts

```diff
@@ -29,6 +29,22 @@ if (process.argv.includes('version')) {
   process.exit(0);
 }
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const os = require('os');
+const clientOS = os.release().toLocaleLowerCase();
+if (clientOS.includes('microsoft') && process.platform === 'linux') {
+  logWarn(`
+**********************************************************
+**********************************************************
+You are using "${yellow(`WLS`)}" as a terminal, if you get an
+error please read the pre-requisites for Microsoft WLS.
+https://scullyio.firebaseapp.com/docs/learn/getting-started/installation/
+**********************************************************
+**********************************************************
+      `);
+}
+
+
 (async () => {
```

Comment:
This warning also pop's up if they configured it correctly, right?
I can live with that. Sort-of.

Reply:
We can not know if have chrome install or not, So, is a warning like the SSL warning